integrations-and-supported-tools/prophet/scripts/Neptune_prophet.py

```diff
@@ -0,0 +1,32 @@
+import neptune.new as neptune
+import neptune_prophet.impl as npt_utils
+import pandas as pd
+from prophet import Prophet
+
+run = neptune.init(
+    project="common/fbprophet-integration",
+    api_token="ANONYMOUS",
+    tags=["fbprophet", "additional regressors", "script"],  # optional
+)
+
+df = pd.read_csv(
+    "https://raw.githubusercontent.com/facebook/prophet/master/examples/example_wp_log_R.csv"
+)
+
+df["cap"] = 8.5
```

Comment:
What is this?

Reply:
That's market size, it's part of the example in FBprophet docs.

https://facebook.github.io/prophet/docs/saturating_forecasts.html#forecasting-growth

I don't have any strong opinions on keeping it, so if it makes it easier for the users, we can remove that specific line.

Reply:
OK, wasn't aware of that. Maybe we could add a code comment to say what it is? It looks strange that this line appears out of the blue in the code.